lib/tasks/generateResourcesJson.js

```diff
@@ -0,0 +1,94 @@
+"use strict";
+
+const resourceListCreator = require("../processors/resourceListCreator");
+
+function getCreatorOptions(projectName) {
+	const creatorOptions = {};
+	if ( projectName === "sap.ui.core" ) {
+		Object.assign(creatorOptions, {
+			externalResources: {
+				"sap/ui/core": [
+					"*",
+					"sap/base/",
+					"sap/ui/"
+				]
+			},
+			mergedResourcesFilters: [
+				"jquery-sap*.js",
+				"sap-ui-core*.js",
+				"**/Component-preload.js",
+				"**/library-preload.js",
+				"**/library-preload-dbg.js",
+				"**/library-preload.json",
+				"**/library-all.js",
+				"**/library-all-dbg.js",
+				"**/designtime/library-preload.designtime.js",
+				"**/library-preload.support.js"
+			].join(",")
```

Comment:
I thought I had commented on this earlier, but can't find it. 

The representation of the filters as comma separated lists is a legacy from the Maven tooling. In a Maven Mojo, those lists have been configured as comma separated lists. But here, in a JavaScript task API, this doesn't make much sense. I would rather switch to pure arrays. 

But as long as this is an internal API (as it is currently), it doesn't matter and we could refactor this later.

Reply:
the rebase -> force push changed the history o0 maybe it was lost/hard to find due to that